### PR TITLE
Clarify how to see your Worker's startup time

### PR DESCRIPTION
**Pull request by irvinebroque**
Explain in the Workers limits docs how to view your Worker's startup time

### CODE REVIEW

File: src/content/docs/workers/platform/limits.mdx [hunk 1]
```diff
@@ -274,6 +274,10 @@ Note that larger Worker bundles can impact the start-up time of the Worker, as t
 
 A Worker must be able to be parsed and execute its global scope (top-level code outside of any handlers) within 400 ms. Worker size can impact startup because there is more code to parse and evaluate. Avoiding expensive code in the global scope can keep startup efficient as well.
 
+You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy`, Wrangler will output the startup time of your Worker in the command-line output, using the [`startup_time_ms` field](/api/operations/worker-script-upload-worker-module) returned by the Cloudflare API.
+
+When you deploy your Worker to Cloudflare using Wrangler, Wrangler returns the startup time
```
Comment (irvinebroque) [inline src/content/docs/workers/platform/limits.mdx]:
```suggestion
```

File: src/content/docs/workers/platform/limits.mdx [hunk 2]
```diff
@@ -274,6 +274,9 @@ Note that larger Worker bundles can impact the start-up time of the Worker, as t
 
 A Worker must be able to be parsed and execute its global scope (top-level code outside of any handlers) within 400 ms. Worker size can impact startup because there is more code to parse and evaluate. Avoiding expensive code in the global scope can keep startup efficient as well.
 
+You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy`, Wrangler will output the startup time of your Worker in the command-line output, using the [`startup_time_ms` field](/api/operations/worker-script-upload-worker-module) returned by the Cloudflare API.
+
```
Comment (irvinebroque) [inline src/content/docs/workers/platform/limits.mdx]:
```suggestion
```

File: src/content/docs/workers/platform/limits.mdx [hunk 3]
```diff
@@ -274,6 +274,8 @@ Note that larger Worker bundles can impact the start-up time of the Worker, as t
 
 A Worker must be able to be parsed and execute its global scope (top-level code outside of any handlers) within 400 ms. Worker size can impact startup because there is more code to parse and evaluate. Avoiding expensive code in the global scope can keep startup efficient as well.
 
+You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy`, Wrangler will output the startup time of your Worker in the command-line output, using the [`startup_time_ms` field](/api/operations/worker-script-upload-worker-module) returned by the Cloudflare API.
```
Comment (irvinebroque) [inline src/content/docs/workers/platform/limits.mdx]: @danlapid — is this just with `wrangler deploy` or also `wrangler versions upload`?

Comment (danlapid) [reply src/content/docs/workers/platform/limits.mdx]: both, Taylor added the versions one, but with versions you still need the experimental flag, 
image

Comment (irvinebroque) [reply src/content/docs/workers/platform/limits.mdx]: Let's add the note about versions and link to it

Comment (danlapid) [inline src/content/docs/workers/platform/limits.mdx]:
```suggestion
You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy` or `npx wrangler@latest versions upload --experimental-versions`, Wrangler will output the startup time of your Worker in the command-line output, using the `startup_time_ms` field in the [Workers Script API](/api/operations/worker-script-upload-worker-module) or [Workers Versions API](/api/operations/worker-versions-upload-version#request-body).
```

**Review by danlapid — APPROVED** [review]

**Review by kodster28 — APPROVED** [review]

**Pull request merged**